packages/webapp/pages/settings/subscription.tsx

```diff
@@ -91,8 +91,12 @@ const PlusInfo = (): ReactElement => {
           tag="a"
           size={ButtonSize.Small}
           variant={ButtonVariant.Secondary}
-          href={managePlusUrl}
-          target="_blank"
+          href={plusHref}
+          target={
```

Comment:
do you need `event.preventDefault()` in iOS case as well?

Reply:
Nah, it is returned as href `undefined` on iOS
https://github.com/dailydotdev/apps/blob/a29ff21ec7e24e3b3cc2ee5499604b85feb7ff56/packages/shared/src/hooks/usePlusSubscription.ts#L46-L48